deployments/multiple-ragapps/docker-compose.yml

```
@@ -54,19 +54,26 @@ services:
       - KEYCLOAK_ADMIN_PASSWORD=admin
     volumes:
       # Mount example realm data
-      - ./keycloak/example/:/opt/keycloak/data/
+      - ${STATE_DIR:-${PWD}/data}/keycloak/example/:/opt/keycloak/data/
     networks:
       - ragapp-network
+    depends_on:
+      - traefik
 
   manager:
     image: ${MANAGER_IMAGE:-ragapp/manager:latest}
     build: ../../src/manager
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock:ro
+      - ${STATE_DIR:-${PWD}/data}/manager:/app/config
+      # Mount the ragapps data folder to the manager
+      - ${STATE_DIR:-${PWD}/data}/ragapps:/app/data/ragapps
     environment:
       - BASE_URL=/manager
       - RAGAPP_IMAGE=${RAGAPP_IMAGE:-ragapp/ragapp:latest}
       - CHAT_REQUEST_LIMIT_THRESHOLD=20
+      # The path in host machine where the ragapps are stored
+      - RAGAPP_MOUNT_PATH=${STATE_DIR:-${PWD}/data}/ragapps
```

```suggestion
      # The path in host machine where the state is stored
      - STATE_HOST_DIR=${STATE_DIR:-${PWD}/data}
```
ideally we can retrieve this value dynamically from the mounted volume (then we don't need to configure it)

---

> ideally we can retrieve this value dynamically from the mounted volume 

I think we can do this by getting the mounted volume path from manager itself, but i'm not sure it's actually needed. We can also config it later in a yaml template then render it by manager (instead hard-code the template)